generated/niscope/session.py

```diff
@@ -2,6 +2,7 @@
 # This file was generated
 import ctypes
 
+from niscope import _converters  # noqa: F401
```

Comment:
We'll have to remember to remove `# noqa: F401`. Can you put a TODO on the same line or will flake8 whine?

Reply:
Added since flake8 didn't care.